Formula/nemu.rb

```diff
@@ -0,0 +1,31 @@
+class Nemu < Formula
+  desc "Ncurses UI for QEMU"
+  homepage "https://github.com/nemuTUI/nemu"
+  url "https://github.com/nemuTUI/nemu/archive/refs/tags/v3.2.0.tar.gz"
+  sha256 "f35b7e5c534bae294fe6df7685b686682145222a954b6914dfdcc17d386fecba"
+  license "BSD-2-Clause"
+
+  depends_on "cmake" => :build
+  depends_on "gettext"
+  depends_on "json-c"
+  depends_on "libarchive"
+  depends_on "ncurses"
+  depends_on "openssl@3"
+  depends_on "pkg-config"
+
+  on_linux do
+    depends_on "libusb"
+    depends_on "sqlite"
+  end
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+  end
+
+  test do
+    expected = /^Config file .* is not found.*$/
+    assert_match expected, shell_output("echo n | XDG_CONFIG_HOME=#{Dir.home} #{bin}/nemu --list")
```

Comment:
Can you try `pipe_output` instead of `shell_output`?

Reply:
Sure, fixed.